### ISSUE TITLE
Select multiple resource and days in timelive view

### ISSUE THREAD

**Issue by GidanMX2** [fullcalendar/fullcalendar-scheduler#310]
Hi,
I have discovered by an answer on my question asked on [stackoverflow](http://stackoverflow.com/questions/43909082/fullcalendar-how-is-possible-to-select-multiple-resource-and-days-in-timelive-v) that select multiple resource for create a multiple resource event is not yet supported.

I think is a good feature to be implemented.

**Comment by arshaw** [comment]
Could you walk me through exactly how the user would create the selection with their mouse/touchpad?

Would they be selecting dates/times within the view, across resources?

Or would they be clicking the name of the resource to "select" it?

**Comment by GidanMX2** [comment]
Yes, they be selecting date/times within the view across resources

**Comment by netalex** [comment]
humm.. i could imagine that a rectangular-like selection on events area will result in a _adiacent_ resources multi-resource event, where instead ctrl(meta)+click _until mouse release_ will result in a non adiacent one.
Could this design work?
as a workaround, in the meantime, i would try this:
intuitively, i would put a popup asking for other resurces to user into the select cb, and manually change the event's resourceId parameter to resourceIds with the array of those other resource's id as value. –

**Comment by flyingangel** [comment]
I would like to have this feature too. Multiple line selecting (to create new event with multiple ressource)

**Comment by kannadasanp** [comment]
Yes, It is a needed one.
![capture](https://user-images.githubusercontent.com/38586030/49875366-d89af580-fe46-11e8-894e-ae782663c7d2.PNG)

Selecting multiple resources in agendaDay view with the time . I dont know whether it is already there in agendaDay view.
But this is a good and needed feature to create a new event with multiple resources at one drag select. 

@arshaw   #310

**Comment by leiferic93** [comment]
Yes, the selection of multiple ressource by dragging across the columns would be great!
